playground/backend/cmd/server/controller.go

```diff
@@ -53,26 +52,26 @@ func (controller *playgroundController) RunCode(ctx context.Context, info *pb.Ru
 	cacheExpirationTime := controller.env.ApplicationEnvs.CacheEnvs().KeyExpirationTime()
 	pipelineId := uuid.New()
 
-	lc, err := setupLifeCycle(info.Sdk, info.Code, pipelineId, controller.env.ApplicationEnvs.WorkingDir())
+	lc, err := setuper.SetupLifeCycle(info.Sdk, info.Code, pipelineId, controller.env.ApplicationEnvs.WorkingDir())
 	if err != nil {
-		return nil, errors.InternalError("Run code", "Error during setup file system: "+err.Error())
+		return nil, errors.InternalError("Run code", "Error during setuper file system: "+err.Error())
```

Comment:
Changed.

Reply:
why you ignore me? :))

Reply:
@ilya-kozyrev it was me, who resolved this conversation :) I thought in your comment you are agreeing with what I've written above, did I misunderstand it?🙂

Reply:
I'm more about package name, in go packages should provide an understanding of the functionality, i.g. json/encoding tell us that we will work with JSON and encoding functionality so that setuper/setuper don't say anything about which functionality it provides.

Reply:
Updated naming.

Reply:
@AydarZaynutdinov I see you also update a structure by creating a corresponding setuper for each structure – nice!